sriov/common/exec.py

```diff
@@ -248,7 +248,7 @@ def execute(self, cmd: str, timeout: int = 5) -> Tuple[int, list, list]:
             shout.pop(0)
         if sherr and echo_cmd in sherr[-1]:
             sherr.pop()
-        if sherr and cmd in sherr[0]:
+        if sherr and len(sherr) > 1 and cmd in sherr[0]:
```

Comment:
how about replace `cmd in sherr[0]` with `shout[0].endswith(cmd + "\n")`

Reply:
typo above, I mean `sherr[0].endswith(cmd + "\n")`; but shout[0] can use the same handling